### DISCUSSION TITLE
Setting position and rotation

### DISCUSSION THREAD

**Discussion by PilotTam** [molstar/molstar#843] [Q&A] [answered]
After digging into the codes for some time I cannot find existing functionality of setting the relative position of a structure like it can be in NGL. If I am to implement this where should I get started, in mol-plugin-state?

**Comment by arose** [comment, marked as answer]
Can you use `TransformStructureConformation`?

https://github.com/molstar/molstar/blob/61a294c889fee1769beded57f34bb569f545ecdd/src/mol-plugin-state/transforms/model.ts#L571-L632

**Reply by PilotTam** [reply]
I see. Thanks! I found  `transform()` from basicWrapper example and wwpdb extension helpful as well.
